core-bundle/contao/classes/BackendTemplate.php

```diff
@@ -88,7 +88,7 @@ private function compile()
 			foreach (array_unique($GLOBALS['TL_JAVASCRIPT']) as $javascript)
 			{
 				$options = StringUtil::resolveFlaggedUrl($javascript);
-				$strJavaScripts .= Template::generateScriptTag($this->addStaticUrlTo($javascript), $options->async, $options->mtime);
+				$strJavaScripts .= preg_replace('/>$/', ' data-turbo-track="dynamic">', Template::generateScriptTag($this->addStaticUrlTo($javascript), $options->async, $options->mtime));
```

Comment:
This cannot work I think as removing scripts won't unload them/remove their listeners/free their memory etc.

Reply:
```suggestion
				$strJavaScripts .= Template::generateScriptTag($this->addStaticUrlTo($javascript), $options->async, $options->mtime);
```

Imo that change is not needed as we are only talking about Styles in the Issue. We would not need to add the attribute to any JS.

Reply:
> To quote the documentation: `Typically used to [remove style and link elements](https://turbo.hotwired.dev/handbook/drive#removing-assets-when-they-change) during navigation.`
> 
> https://turbo.hotwired.dev/handbook/drive#removing-assets-when-they-change
> 
> I do think it's the right fix but we should not add `dynamic` to JS.

@zoglo it says "typically used". Turbo just makes you aware how JavaScript needs to work with Turbo and wants to make sure that you don't get a misconception about this.